amazon_sqs_receive_message_block.py

```diff
@@ -29,12 +29,12 @@ class SQSReceiveMessage(SQSBase):
         # ^ FIFO queues only
 
     def process_signals(self, signals):
+        new_signals = []
         for signal in signals:
             try:
                 self.logger.debug("Sending message via {} queue".format(
                     self.queue_url(signal)))
-
```

Comment:
I don't know anything about this library, but this doesn't seem right to me. Is `receive_message` supposed to be in `process_signals`? Seems more likely that it should be in a loop that continuously tries to receive messages.